plugins/extractors/redshift/redshift.go

```diff
@@ -0,0 +1,267 @@
+package redshift
+
+import (
+	"context"
+	_ "embed" // used to print the embedded assets
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/redshiftdataapiservice"
+	"github.com/aws/aws-sdk-go/service/redshiftdataapiservice/redshiftdataapiserviceiface"
+	"github.com/odpf/meteor/models"
+	commonv1beta1 "github.com/odpf/meteor/models/odpf/assets/common/v1beta1"
+	facetsv1beta1 "github.com/odpf/meteor/models/odpf/assets/facets/v1beta1"
+	assetsv1beta1 "github.com/odpf/meteor/models/odpf/assets/v1beta1"
+	"github.com/odpf/meteor/plugins"
+	"github.com/odpf/meteor/registry"
+	"github.com/odpf/meteor/utils"
+	"github.com/odpf/salt/log"
+	"strings"
+)
+
+//go:embed README.md
+var summary string
+
+var defaultExcludes = []string{"information_schema", "pg_catalog", "pg_internal", "public"}
```

Comment:
these exclude list is postgres specific, I am not sure it fits within redshift context

Reply:
In db querying, redshift works in similar way as postgres,  [the list here is redshift specific](https://docs.aws.amazon.com/redshift/latest/mgmt/data-api.html) but yes till we test it with real cluster data we can remove it.